### DISCUSSION TITLE
Standard public folder ?

### DISCUSSION THREAD

**Discussion by BestNoob** [rejetto/hfs#733] [Q&A]
So if i understand it correctly there is a config file for every folder you add?
Is there a standard folder after setup where you just can public visible add files to ?

Running Ubuntu Server

**Comment by rejetto** [comment]
you only have 1 config file.
you don't need to mess with it anyway, you configure things in the Admin-panel.
By default everything you add is public to download while other operations (upload, delete) are (by default) for no one.
